sympy/physics/vector/tests/test_point.py

```diff
@@ -126,3 +128,67 @@ def test_point_partial_velocity():
     assert p.partial_velocity(N, u1) == A.x
     assert p.partial_velocity(N, u1, u2) == (A.x, N.y)
     raises(ValueError, lambda: p.partial_velocity(A, u1))
+
+def test_point_vel():
+    q1, q2 = dynamicsymbols('q1 q2')
+    N = ReferenceFrame('N')
+    B = ReferenceFrame('B')
+    S = ReferenceFrame('S')
+    O = Point('O')
+    Q = Point('Q')
+    Q.set_pos(O, q1 * N.x)
+    raises(ValueError , lambda : Q.vel(N)) #Velocity of Q is not defined
+    O.set_vel(N, q2 * N.y)
+    assert Q.vel(N) == Derivative(q1) * N.x + q2 * N.y  # Velocity of Q using O
+    P1 = Point('P1')
+    P1.set_pos(O, q1 * B.x)
+    raises(ValueError, lambda : P1.vel(B)) # O's velocity is defined in different frame
+    raises(ValueError, lambda : P1.vel(N)) # Pos vector is defined in different frame
```

Comment:
You set the velocity of O in N and the position from O to P1, so shouldn't `P1.vel(n)` work?

Reply:
The position vector is not defined in that frame